pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php

```diff
@@ -1199,6 +1212,18 @@ public function modal_content_fields_pickup_request() {
 		}
 	}
 
+	protected function is_UK_shipment( $order_id ) {
+		$order = wc_get_order( $order_id );
+		$shipping_address = $order->get_address( 'shipping' );
+		$shipping_country = $shipping_address['country'];
+
+		if( $shipping_country == "GB" ) {

```

Comment:
let's start using strict comparison.
```suggestion
		if ( 'GB' === $shipping_country ) {

```